src/table/headerRenderers/dataHeader.component.tsx

```diff
@@ -81,14 +83,21 @@ const DataHeader = (props: DataHeaderProps): React.ReactElement => {
       direction={currSortDirection}
       onClick={() => onSort(dataKey, nextSortDirection)}
     >
-      <Typography noWrap sx={{ fontSize: 'inherit', lineHeight: 'inherit' }}>
+      {columnIconMappings.get(dataKey.toUpperCase()) ?? <FeedIcon />}
```

Comment:
If we're defining the Icons like this (aka via a global map & lookup via `dataKey`) - then isn't the `icon` prop obsolete? If so we should remove the code relating to the `icon` prop method.

Reply:
That was my bad for missing that... see my latest commit, which uses the correct method. Keep an eye on snapshot changes